docs/tutorials/clickstream/index.rst

```diff
@@ -1,7 +1,7 @@
-.. _ksql_clickstream:
+.. _ksql_clickstream-overview:
 
-Clickstream Analysis Demo
-=========================
+Clickstream Analysis Tutorial
```

Comment:
shouldn't the naming of this be consistent with the docker version - which is called 'Clickstream Analysis using Docker' - so perhaps call it - Clickstream analysis?

Reply:
Yes, i have trimmed off `tutorial`

Reply:
@bluemonk3y What do you think about `Clickstream Data Analysis Pipeline (Docker)` and `Clickstream Data Analysis Pipeline (Local)`?